content/smart.md

```diff
@@ -0,0 +1,140 @@
+
+What technology and framework you use is really up to you and your company. Regardless of which technology, framework, or language you choose, make sure that your app is fast, responsive, and adaptable to various screen sizes.  We recently created [ASCVD Risk SMART app][ASCVD-RISK-APP] using React.js.
+
+### Open Source FHIR Client Libraries ###
+
+To start development quickly, there is an open source [fhir-client JavaScript library][FHIR-CLIENT-JS] that takes care of the OAuth2 handshake and provides a built-in library to call FHIR resources. The library is usable but has several known issues in previous versions. One particular issue is around the usage of the `sessionStorage` property. Cerner highly recommends that you upgrade the library to version [v0.1.10][FHIR-CLIENT-v0.1.10] or higher to correct a known patient safety issue. You also need to download and include this [additional code][FHIR-CLIENT-CERNER-ADDITIONS] into your project to correct this issue.
+
+Other additional FHIR clients are available:
+
+- Java: [http://hapifhir.io/doc_rest_client.html][FHIR-CLIENT-JAVA]
+- .NET: [https://github.com/ewoutkramer/fhir-net-api][FHIR-CLIENT-NET]
+- Python: [https://github.com/smart-on-fhir/client-py][FHIR-CLIENT-PYTHON]
+- iOS: [https://github.com/smart-on-fhir/Swift-FHIR][FHIR-CLIENT-IOS]
+
+### Tutorial ###
+
+Creating your first app can be overwhelming, so Cerner has created a [step by step tutorial][APP-TUTORIAL] on how to start creating your very first SMART app.  The app is written in HTML and JavaScript with some CSS.  We highly recommend that you take a look at the tutorial.
+
+### Code Console and Sandbox ###
+
+Cerner's implementation of SMART on FHIR ecosystem allows you to register, update, and delete your SMART app from [Code Console][CODE-CONSOLE].  Additionally, you can open and test your app from Code Console in our Sandbox environment.  Currently, there is a 15-minute waiting period for new or updated app configurations to take effect.  If you cannot open the application after the 15-minute period, post a question to the [Cerner FHIR Developers Google Group][CERNER-FHIR-DEV-GOOGLE-GROUP].
+
+### Launch Context ###
+
+See the [Scopes and Launch Content][SMART-HEALTH-IT-SCOPES-LAUNCH-CONTEXT] page for more information on how you can obtain the context. Generally, your app must able to retrieve SMART context parameters such as patient id, encounter id (if available), tenant, and a few others. You can inspect the OAuth2 access_token value that is returned to see the actual context.
+
+## Testing ##
+
+### Code Console ###
+
+For provider and patient facing apps, you can select any patient from a list of patients in our Sandbox environment to begin testing. After selecting a patient to test, you can open your app from a new web browser in [Code Console][CODE-CONSOLE]. Testing from Code Console through the browser helps you start developing and testing your app quickly.
+
+### PowerChart ###
```

Comment:
We can but I'd need to get more info from @torresdrew as I'm not aware of "Patient Portal" is being used for testing by developers.